test/test_integration_policy.py

```diff
@@ -116,6 +116,13 @@ def test_policy_bad_syntax_raises_error(self):
         self.assertIn("422 Unprocessable Entity {\"error\":{\"code\":\"validation_failed\",\"message\":",
                   str(mock_log.output))
 
+    @integration_test()
+    def test_policy_bad_syntax_raises_error_to_terminal(self):
+        policy = "- ! user bad syntax"
```

Comment:
can you explain the test its not clear from here ? it will fail on file not exist , no?